frontend/src/pages/HouseholdForm.js

```diff
@@ -21,6 +22,10 @@ const HouseholdFormPage = () => {
 
   return (
     <div>
+      <Helmet>
+        <title>Passenger vehicle rebate application form – CleanBC Go Electric</title>
+      </Helmet>
+      Hello BCeID {decoded.preferred_username}
```

Comment:
@vibhiquartech I think this decoded part might have been from the rebase, this variable doesn't exist anymore